src/transform/transform.ts

```diff
@@ -584,13 +583,19 @@ function createResponseType(codeModel: CodeModel, group: OperationGroup, op: Ope
         const pager = {
           name: name,
           op: op,
+          respField: false,
```

Comment:
`respField: isLROOperation(op),` then remove the below `if` block.

Reply:
Done